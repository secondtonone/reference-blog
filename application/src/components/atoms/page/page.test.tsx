import renderer from 'react-test-renderer';
import Page from '.';

describe('Match snapshot Page', () => {
  it('default', () => {
    const tree = renderer.create(<Page />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
