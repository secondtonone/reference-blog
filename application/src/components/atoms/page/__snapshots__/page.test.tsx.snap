// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Match snapshot Page default 1`] = `
<div
  className="page__PageStyled-sc-1uolz5z-0 fLUSeG"
  data-test="page"
/>
`;
